src/site/content/en/blog/native-file-system/index.md

```diff
@@ -246,27 +312,32 @@ butDir.addEventListener('click', async (e) => {
 });
 ```
 
-## What's currently supported? {: #whats-supported }
-
-We're still working on some of the implementation for the
-Native File System API, and not everything in the [spec][spec]
-(or [explainer][explainer]) has been completed.
-
-As of Chrome 78, the following functionality is not available, or
-doesn't match the spec:
-
-* Handles are not serializable, meaning they cannot be passed via
-  `postMessage()`, or stored in IndexedDB.
-* Non-atomic writes (i.e. calls to `FileSystemFileHandle.createWriter()`
-  with `inPlace: true`).
-* Writing to a file using a [`WritableStream`][writablestream].
-* The [`FileSystemDirectoryHandle.resolve()`][fs-dir-handle] method.
+## What's new/changed? {: #whats-new }
+
+Chrome 82
+
+* Support for [writable streams](#save-to-disk) was added, and the previous
+  method for writing to disk (`FileSystemWriter`) was removed.
+* File handles can now be serialized and stored in IndexedDB, or sent via
```

Comment:
IndexedDB again.

Reply:
I believe IndexedDB is the correct term - from MDN https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API, or is there something else?